src/jit/emitxarch.cpp

```diff
@@ -7785,6 +7813,32 @@ void emitter::emitDispIns(
             break;
         }
 
+        case IF_AWR_RRD_CNS:
+        case IF_MWR_RRD_CNS:
+        {
+            assert(ins == INS_vextracti128 || ins == INS_vextractf128);
+            sstr = codeGen->genSizeStr(EA_ATTR(16));
```

Comment:
Why are we doing `sstr` here, I think everyone else is getting it above.

Reply:
`vextracti/f128` is 256-bit instruction that extracts a **128-bit** value, so we have to specially treat the `sstr`.

Reply:
Comments are useful for special comments are useful for special cases.